types/wx-js-sdk-dt/index.d.ts

```diff
@@ -0,0 +1,468 @@
+// Type definitions for wx-js-sdk 1.2
+// Project: https://res.wx.qq.com/open/js/jweixin-1.2.0.js
+// Definitions by: Bian Zhongjie <https://github.com/agasbzj>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export namespace wx {
```

Comment:
Ok, I've changed export to declare. But still
```
Error: 'wx-js-sdk-dt/wx-js-sdk-dt-tests.ts' unnecessarily references the package. This can be removed.
    at Object.<anonymous> (/Users/Agas/DefinitelyTyped/node_modules/types-publisher/src/lib/module-info.ts:383:11)
    at next (native)
    at fulfilled (/Users/Agas/DefinitelyTyped/node_modules/types-publisher/bin/lib/module-info.js:4:58)
npm ERR! Test failed.  See above for more details.
```
Hum...what's wrong with that. It makes npm test fail.